src/platform_impl/macos/system_tray.rs

```diff
@@ -75,6 +81,13 @@ impl SystemTrayBuilder {
       let tray_target: id = msg_send![tray_target, init];
       let _: () = msg_send![button, setAction: sel!(click:)];
       let _: () = msg_send![button, setTarget: tray_target];
+      let _: () = msg_send![
+        button,
+        sendActionOn: NSEventMask::NSLeftMouseDownMask
+          | NSEventMask::NSRightMouseDownMask
+          | NSEventMask::NSKeyDownMask
+          | NSEventMask::NSKeyDownMask
```

Comment:
is this duplication necessary or accident ?

Reply:
oooops! thanks